app/styles/mixins/_platform.scss

```diff
@@ -41,25 +41,3 @@
     @content;
   }
 }
-
-// A mixin which takes a content block that should only
-// be applied when the current (real or emulated) operating
-// system is Linux.
-//
-// This helper mixin is useful in so far that it allows us
-// to keep platform specific styles next to cross-platform
-// styles instead of splitting them out and possibly forgetting
-// about them.
-@mixin linux {
-  body.platform-linux & {
-    @content;
-  }
-}
-
-// An exact copy of the linux mixin except that it allows for
-// writing base-level rules
-@mixin linux-context {
-  body.platform-linux {
-    @content;
-  }
-}
```

Comment:
While how the titlebar is styled doesn't matter to me either way, you'll likely still have some linux specific styling to apply (see my PR on styling scrollbars for dark mode), so I'd recommend keeping these mixins

Reply:
@jfgordon2 good point, but for now I'm going to leave this revert as-is as it makes rebasing to drop the original commit easier. I'll add the theming mixins in a follow-up PR that should support your work in #240.